datafusion/common/Cargo.toml

```diff
@@ -40,13 +40,13 @@ pyarrow = ["pyo3", "arrow/pyarrow"]
 
 [dependencies]
 apache-avro = { version = "0.14", default-features = false, features = ["snappy"], optional = true }
-arrow = { workspace = true, default-features = false }
-arrow-array = { version = "35.0.0", default-features = false, features = ["chrono-tz"] }
```

Comment:
What is the reason for removing `features = ["chrono-tz"]` here? Is this something new in version 35?

Reply:
Never mind, I see this is moved to the workspace.package import